### PR TITLE
CICD Troubleshooting about RestorePackagesWithLockFile

### PR DESCRIPTION
**Pull request by jesp209i**
## Description

Added a section to the CICD flow troubleshooting

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Cloud / CICD Flow

## Deadline (if relevant)

### CODE REVIEW

**Comment by jonat123** [comment]
@jesp209i Thanks for the PR 😄

We will take a look at it ASAP!

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
 # Troubleshooting
+## Special Cases
```
Comment (alina-tincas) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:2]:
```suggestion
# Troubleshooting

## Special Cases
```

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+
+### Using RestorePackagesWithLockFile in your CSPROJ-file
+In instances where people used the `RestorePackagesWithLockFile` set to true, we saw that no changes were made to the website. This happened even though the CI/CD-deployments were completed successfully, and files were updated as expected in the Cloud-repository.
+
+The reason was that the KUDU-deploy process failed. This process takes the newly committed files from the cloud repository and runs restore, build, and publish on the cloud environment.
+
+Removing the `RestorePackagesWithLockFile` allowed deployments to come all the way through.
```
Comment (alina-tincas) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:9]:
```suggestion
### Using `RestorePackagesWithLockFile` in your .csproj file
If `RestorePackagesWithLockFile` is used and set to true, you will experience that no changes will be made to the website. This happened even though the CI/CD deployments were completed successfully, and files were updated as expected in the Cloud repository.

The reason for this is that the KUDU deploy process fails. This process takes the newly committed files from the cloud repository and runs restore, build, and publish on the cloud environment.

To resolve this issue, remove the `RestorePackagesWithLockFile` to allow the deployments to go through as expected.
```

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Hi @jesp209i thank you for the PR 🙌 

I have rephrased some of the content, otherwise all looks good 💪 Mergin this in!

**Pull request merged**